doc/development.md

```diff
@@ -76,167 +79,124 @@ active context
+        you identify the catalog entry name from other operators and the official
+        APIcast operator entries. For example, to `MyOrg`
+  * Edit the `bundle.Dockerfile` file and change the value of
+    the Dockerfile label `LABEL operators.operatorframework.io.bundle.package.v1`
+    to a different value than `apicast-operator`. For example to
+    `myorg-apicast-operator`
+  * Edit the `bundle/metadata/annotations.yaml` file and change the value of
+    `.annotations.operators.operatorframework.io.bundle.package.v1` to a
+    different value than `apicast-operator`. For example to
+    `myorg-apicast-operator`. The new value should match the
+    Dockerfile label `LABEL operators.operatorframework.io.bundle.package.v1`
+    in the `bundle.Dockerfile` as explained in the point above
+
+  It is really important that all the previously shown fields are changed
+  to avoid overwriting the APIcast operator official public operator
+  catalog entry in your cluster and to avoid confusion having two equal entries
+  on it.
+
+* [Create an operator bundle image](#(re)Generate-operator-bundle) using the
+  changed contents above
+
+* [Push the operator bundle into an external container repository](#push-an-operator-bundle-into-an-external-container-repository).
+
+* Run the following command to deploy the operator in your currently configured
+  and active cluster in $HOME/.kube/config:
+  ```sh
+  operator-sdk run bundle --namespace <mynamespace>
+  ```
+
+  Additionally, a specific kubeconfig file with a desired Kubernetes
+  configuration can be provided too:
+  ```sh
+  operator-sdk run bundle --namespace <mynamespace> --kubeconfig <path>
+  ```
 
 It will take a few minutes for the operator to become visible under
 the _OperatorHub_ section of the OpenShift console _Catalog_. It can be
 easily found by filtering the provider type to _Custom_.
 
 [git_tool]:https://git-scm.com/downloads
 [operator-sdk]:https://github.com/operator-framework/operator-sdk
+[docker]:https://docs.docker.com/install/
 [go]:https://golang.org/
 [kubernetes]:https://kubernetes.io/
 [kubectl]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
 
 ### Run tests
 
-#### Run unittests
+#### Run all tests
 
-No access to a Kubernetes cluster is required
+Access to a Kubernetes v1.11.3+ cluster required
 
 ```sh
-make test-crds
+make test
 ```
 
-#### Run integration tests
+#### Run unit tests
 
-Access to a Kubernetes v1.11.0+ cluster required
-
-* Run tests locally deploying image
-```sh
-export NAMESPACE=operator-test
-make e2e-run
-```
+No access to a Kubernetes cluster is required
 
-* Run tests locally running operator with go run instead of as an image in the cluster
 ```sh
-export NAMESPACE=operator-test
-make e2e-local-run
+make test-unit
 ```
 
-## Manifest management
+#### Run end-to-end tests
 
-`operator-courier` is used for metadata syntax checking and validation.
-This can be installed directly from pip:
+Access to a Kubernetes v1.11.3+ cluster required.
```

Comment:
Check requirement

Reply:
It is not strictly needed. I'll remove it :+1:

Reply:
Fixed.